src/main/java/org/gridsuite/securityanalysis/server/entities/ConstraintEntity.java

```diff
@@ -0,0 +1,42 @@
+package org.gridsuite.securityanalysis.server.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Entity
+@Table(name = "contingency_limit_violation_constraint")
+public class ConstraintEntity {
```

Comment:
Rename by `SubjectLimitViolationsEntity` ?
`Constraint` is not a powsybl term, remove it everywhere

Reply:
done